examples/analog/main.cpp

```diff
@@ -83,12 +82,12 @@ int main(int argc, char* argv[])
 	{
 		cout<<val<<endl;
 	}
-	cout << "CHANNEL2"<<endl;
+/*	cout << "CHANNEL2"<<endl;
```

Comment:
this code looks commented out

Reply:
this file is also committed by accident, sorry